database/migrations/platform/2024_04_03_000000_update_player_sessions.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+declare(strict_types=1);
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+return new class() extends Migration {
+    public function up(): void
+    {
+        Schema::table('player_sessions', function (Blueprint $table) {
+            $table->string('user_agent', 255)->nullable()->after('duration');
+            $table->string('ip_addr', 40)->nullable()->after('user_agent');
```

Comment:
nit: I can't find anything else in our schemas that's truncated. Can we name this `ip_address` instead of `ip_addr`?

Reply:
done